packages/table/src/V11Adapter/V11Adapter.tsx

```diff
@@ -127,6 +152,8 @@ const V11Adapter = <T extends LGRowData>({
               virtualRow={
                 useVirtualScrolling ? (iterable as VirtualItem) : undefined
               }
+              // @ts-expect-error rowProps is an additional prop passed by the `processData` function
```

Comment:
why do we need to ts-expect-error here?

Reply:
We should fix the types in the `processData` function rather than add expect error

Reply:
agreeed